ext/opentelemetry-ext-zipkin/src/opentelemetry/ext/zipkin/__init__.py

```diff
@@ -212,6 +212,17 @@ def _extract_tags_from_span(attr):
     return tags
 
 
+def _extract_tags_from_span(span: Span):
+    tags = _extract_tags_from_dict(span.attributes)
```

Comment:
there are cases where a DefaultSpan is used instead of an sdk.trace.Span. In those cases this will return an attributeerror as "attributes" doesn't exist.

There may need to be some sort of getter for attributes in the API itself to ensure this works as expected.

Reply:
Can you elaborate on how I should handle this?
Currently I kept the existing behavior.

Reply:
Right, this was a bug that existed in the current implementation of the exporter. You'd want to use something like `getattr(span, "attributes", None)` to prevent an exception from being raised.

Reply:
Thanks for clarifying! Changed to `getattr`.